Fancy Hats/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>
+            Fancy Hats
+        </title>
+        <script src="https://kit.fontawesome.com/d95ab1ac46.js"></script>
+        <link rel="stylesheet" href="/Fancy Hats/assets/css/styles.css">
+    </head>
+    <body>
+        <!-- START : WHOLE CONTAINER -->
+        <section class="contentContainer">
+
+            <!-- START : NAV SECTION -->
+            <div class="navSection">
+                <div class="hamburgerIcon">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </div>
+                <div class="dropDown">
+                    <ul>
+                        <li><img src="/Fancy Hats/assets/Images/hat-icon.svg" alt="" /></li>
+                        <li>COWBOY HATS</li>
+                        <li>DAD HATS</li>
+                        <li>FEDORAS</li>
+                        <li>TRIBAL HATS</li>
+                        <li>FLAT CAPS</li>
+                        <li>DERBY HATS</li>
+                        <li>ARMY CAPS</li>
+                        <li>CLOCHE HATS</li>
+                        <li>BEAN IES</li>
+                        <li>TRAPPER HATS</li>
+                        <li>SNAP BACKS</li>
+                        <li>TRUCKER HATS</li>
+                        <li>VISORS</li>
+                    </ul>
+                </div>
+            </div>
+            <!-- END : NAV SECTION -->
+            
+            <!-- START : MIDDLE SECTION -->
+            <div class="middleSection">
+                <h2>FANCY HATS</h2>
+                <div class="quoteAboutHat">
+                    <h1 class="quoteFirstLine">HATS FOR A</h1>
+                    <h1 class="quoteSecondLine">LIFE TIME</h1>
+                </div>
+            </div>
+            <div class="yearSection">
+                <div class="yearContainer">
+                    <h4>SINCE &nbsp;</h4><h1>1867</h1>
```

Comment:
why use  ` &nbsp; ` ? go with padding or margin

Reply:
Padding added and &nbsp; removed.